src/core/multifeaturelistmodel.h

```diff
@@ -82,14 +90,24 @@ class MultiFeatureListModel : public QSortFilterProxyModel
      */
     int selectedCount() const;
 
-    void checkSelectedCount();
-
     //! Returns TRUE if the selected features can have their attributes value changed
     bool canEditAttributesSelection();
 
+    //! Returns TRUE if the selected features can be merged
+    bool canMergeSelection();
+
     //! Returns TRUE if the selected features can be deleted
     bool canDeleteSelection();
 
+    /**
+     * Merges selected features by updating the first seleted feature's geometry
+     * to a combinasion (i.e. union) of geometries of all selected features.
```

Comment:
```suggestion
     * to a combination (i.e. union) of geometries of all selected features.
```

combinaison?

Reply:
My head was evidentially having an internal fight between French and English, must have been a draw :wink: